textproto/header.go

```diff
@@ -396,6 +421,19 @@ func trimAroundNewlines(v []byte) string {
 	return b.String()
 }
 
+const (
+	MaxHeaderLines = 1000
```

Comment:
Can we keep these unexported? If we were to allow users to customize these limits, I'd prefer not to make this global. I'd also prefer users not to rely on these values.

Reply:
Fair point.